resources/helm/conda-store/values.yaml

```diff
@@ -0,0 +1,176 @@
+gateway:
+  replicas: 1
+
+  annotations: {}
+
+  resources: {}
+
+  prefix: /
+
+  loglevel: INFO
+
+  image:
+    name: quansight/conda-store-server
+    tag: "set-by-chartpress"
+    pullPolicy: IfNotPresent
+
+  imagePullSecrets: []
+
+  service:
+    annotations: {}
+
+  auth:
+    type: simple
+
+    simple:
+      password:
+
```

Comment:
As this will work as the default value template conda-store will look into it when deployed and as a reference point. I would suggest including comments describing its field and its optional values. 

As a ref. https://github.com/PrefectHQ/server/blob/master/helm/prefect-server/values.yaml